[BUG] Compile error with HOST_ACTION + G29_RETRY_AND_RECOVER

<!--

Please follow the instructions below. Failure to do so may result in your issue being closed.

### Before Reporting a Bug

1. Test with the `bugfix-2.0.x` branch to see whether the issue still exists.

2. Get troubleshooting help from the Marlin community to confirm it's a bug and not just a configuration error. Links at https://github.com/MarlinFirmware/Marlin/issues/new/choose

### Instructions

1. Fill out every section of the template below.

2. Always attach configuration files, regardless of whether you think they are involved.

3. Read and understand Marlin's Code of Conduct. By filing an Issue, you are expected to comply with it, including treating everyone with respect: https://github.com/MarlinFirmware/Marlin/blob/master/.github/code_of_conduct.md

-->

### Bug Description

Unable to compile 2021-01-22 commit version (no issue with 2021-01-18)
In src/gcode/gcode.cpp, host_action (line 215 / 224) and host_action_cancel (line 231) isn't recognized

G29_RETRY_AND_RECOVER And  HOST_ACTION enabled

### Configuration Files

[Configuration.zip](https://github.com/MarlinFirmware/Marlin/files/5856891/Configuration.zip)

### Steps to Reproduce

Using Vscode with PlatformIO (updated), and last commit (2021-01-22),
using default config, modify it :
in configuration.h :
- enable BL_TOUCH
- enable AUTO_BED_LEVELING_BILINEAR
- enable Z_SAFE_HOMING
in configuration_adv.h :
- enable HOST_ACTION_COMMAND
- enable G29_RETRY_AND_RECOVER

and compile

**Expected behavior:**
To compile without error

**Actual behavior:**

Error at compilation : 

Compiling .pio\build\mega2560\src\src\gcode\host\M110.cpp.o
Marlin\src\gcode\gcode.cpp: In static member function 'static void GcodeSuite::event_probe_recover()':
Marlin\src\gcode\gcode.cpp:215:46: error: 'host_action' was not declared in this scope
       host_action(PSTR(ACTION_ON_G29_RECOVER));
                                              ^
Marlin\src\gcode\gcode.cpp: In static member function 'static void GcodeSuite::event_probe_failure()':
Marlin\src\gcode\gcode.cpp:224:46: error: 'host_action' was not declared in this scope
       host_action(PSTR(ACTION_ON_G29_FAILURE));
                                              ^
Marlin\src\gcode\gcode.cpp:231:28: error: 'host_action_cancel' was not declared in this scope
         host_action_cancel();
                            ^
At global scope:
cc1plus.exe: warning: unrecognized command line option '-Wno-register'
*** [.pio\build\mega2560\src\src\gcode\gcode.cpp.o] Error 1

#### Additional Information

--- a/Marlin/src/gcode/gcode.cpp
+++ b/Marlin/src/gcode/gcode.cpp
@@ -40,7 +40,7 @@ GcodeSuite gcode;
   #include "../module/printcounter.h"
 #endif
 
-#if ENABLED(HOST_PROMPT_SUPPORT)
+#if ENABLED(HOST_ACTION_COMMANDS)
   #include "../feature/host_actions.h"
 #endif
 

